schutzbot/deploy.sh

```diff
@@ -89,9 +102,25 @@ fi
 if [ -f "rhel8internal.repo" ]; then
     greenprint "Preparing repos for internal build testing"
     sudo mv rhel8internal.repo /etc/yum.repos.d/
-    sudo rm -f /etc/yum.repos.d/osbuild*.repo
+    # Use osbuild from schutzfile if desired for testing custom osbuild-composer packages
+    # specified by $REPO_URL in ENV and used in prepare-rhel-internal.sh
+    if [ "$SCHUTZ_OSBUILD" == 1 ]; then
+        sudo rm -f /etc/yum.repos.d/osbuild-composer.repo
+    else
+        sudo rm -f /etc/yum.repos.d/osbuild*.repo
+    fi
```

Comment:
This is again related to nightly testing only. Doesn't need to be here. I guess it got here along with some other changes. It's fine if you leave it as is though I think...